flocker/control/test/test_persistence.py

```diff
@@ -3,31 +3,40 @@
 """
 Tests for ``flocker.control._persistence``.
 """
+import json
+import string
 
-from uuid import uuid4
+from uuid import uuid4, UUID
 
 from eliot.testing import validate_logging, assertHasMessage, assertHasAction
 
+from hypothesis import given
+from hypothesis import strategies as st
+
 from twisted.internet import reactor
 from twisted.trial.unittest import TestCase, SynchronousTestCase
 from twisted.python.filepath import FilePath
 
-from pyrsistent import PRecord
+from pyrsistent import PRecord, pset
 
 from .._persistence import (
     ConfigurationPersistenceService, wire_decode, wire_encode,
-    _LOG_SAVE, _LOG_STARTUP, LeaseService,
+    _LOG_SAVE, _LOG_STARTUP, LeaseService, migrate_configuration,
+    _CONFIG_VERSION, ConfigurationMigration, ConfigurationMigrationError,
+    _LOG_UPGRADE, MissingMigrationError,
     )
 from .._model import (
     Deployment, Application, DockerImage, Node, Dataset, Manifestation,
-    AttachedVolume, SERIALIZABLE_CLASSES, NodeState)
+    AttachedVolume, SERIALIZABLE_CLASSES, NodeState, Configuration,
+    Port, Link,
+    )
 
 
-DATASET = Dataset(dataset_id=unicode(uuid4()),
+DATASET = Dataset(dataset_id=u'4e7e3241-0ec3-4df6-9e7c-3f7e75e08855',
                   metadata={u"name": u"myapp"})
 MANIFESTATION = Manifestation(dataset=DATASET, primary=True)
 TEST_DEPLOYMENT = Deployment(
-    nodes=[Node(uuid=uuid4(),
+    nodes=[Node(uuid=UUID(u'ab294ce4-a6c3-40cb-a0a2-484a1f09521c'),
                 applications=[
                     Application(
                         name=u'myapp',
```

Comment:
I'm guessing that the reason behind this change is so that these variables match the sample data files added in the PR?

If so, please add a comment explaining that these values were not chosen at random, but that they must match those files.